benchmarks/python/test_rmsnorm_add_fwd.py

```diff
@@ -30,3 +70,35 @@ def test_rmsnorm_add_fwd_baseline_benchmark(
         benchmark_fn,
         [inputs, weights, residual],
     )
+
+
+# flash infer does inplace update of inputs and residual tensors
+# needs to explicitly return the updated tensors to correctly compute IO bytes
+# https://github.com/flashinfer-ai/flashinfer/blob/ba2b4aa636c4ecf99981794767ffbf89267720cd/flashinfer/norm.py#L106
+def flashinfer_rmsnorm_add_wrapper(inputs_list):
+    inputs, weights, residual = inputs_list
+    fused_add_rmsnorm(inputs, residual, weights)
+    return inputs, residual
+
+
+# requires flashinfer to be installed
+# `pip install flashinfer-python` works in pjnl container
+@pytest.mark.parametrize("executor", ["flashinfer"])
+@pytest.mark.parametrize("size", generate_input_sizes_rmsnorm_add(dims=2))
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES)
+@pytest.mark.inner_persistent
+def test_rmsnorm_add_fwd_flashinfer_benchmark(
```

Comment:
Why are we not adding it to the above function which runs all other executors?

Reply:
I consider it a special benchmark requires installing flash infer and wraper for api. 
I also feel too early to add it to `DEFAULT_EXECUTORS` since we just started testing it for this specific case.
We can make this change if more and more cases are being benchmarked uisng flash infer.

Reply:
No need to add it to the DEFAULT_EXECUTORS, you can simply append it to the list of executors. For `flash_infer`, instead of using `with_executor`, the wrapper for `fused_rmsnorm` is used.

> We can make this change if more and more cases are being benchmarked uisng flash infer.

That would become a larger change in my opinion, hence my suggestion.
I'll leave it to you to decide.

Reply:
sounds a good idea. merged two tests using `DEFAULT_EXECUTORS + ["flashinfer"]`